hexapod_controller:
  cycle_time: 1.0
  standing_height: 0.15
  step_height: 0.05
  leg_x_offset: 0.1
  leg_y_offset: 0.1



# Publish all joint states -----------------------------------
joint_state_controller:
  type: joint_state_controller/JointStateController
  publish_rate: 50

# Position Co3rollers --------------------------------------
hip_joint_1_position_controller:
  type: effort_controllers/JointPositionController
  joint: hip_joint_1
  pid: {p: 7.0, i: 0.0, d: 0.2}

hip_joint_3_position_controller:
  type: effort_controllers/JointPositionController
  joint: hip_joint_3
  pid: {p: 7.0, i: 0.0, d: 0.2}

hip_joint_5_position_controller:
  type: effort_controllers/JointPositionController
  joint: hip_joint_5
  pid: {p: 7.0, i: 0.0, d: 0.2}

hip_joint_2_position_controller:
  type: effort_controllers/JointPositionController
  joint: hip_joint_2
  pid: {p: 7.0, i: 0.0, d: 0.2}

hip_joint_4_position_controller:
  type: effort_controllers/JointPositionController
  joint: hip_joint_4
  pid: {p: 7.0, i: 0.0, d: 0.2}

hip_joint_6_position_controller:
  type: effort_controllers/JointPositionController
  joint: hip_joint_6
  pid: {p: 7.0, i: 0.0, d: 0.2}

knee_joint_1_position_controller:
  type: effort_controllers/JointPositionController
  joint: knee_joint_1
  pid: {p: 7.0, i: 0.0, d: 0.2}

knee_joint_3_position_controller:
  type: effort_controllers/JointPositionController
  joint: knee_joint_3
  pid: {p: 7.0, i: 0.0, d: 0.2}

knee_joint_5_position_controller:
  type: effort_controllers/JointPositionController
  joint: knee_joint_5
  pid: {p: 7.0, i: 0.0, d: 0.2}

knee_joint_2_position_controller:
  type: effort_controllers/JointPositionController
  joint: knee_joint_2
  pid: {p: 7.0, i: 0.0, d: 0.2}

knee_joint_4_position_controller:
  type: effort_controllers/JointPositionController
  joint: knee_joint_4
  pid: {p: 7.0, i: 0.0, d: 0.2}

knee_joint_6_position_controller:
  type: effort_controllers/JointPositionController
  joint: knee_joint_6
  pid: {p: 7.0, i: 0.0, d: 0.2}

ankle_joint_1_position_controller:
  type: effort_controllers/JointPositionController
  joint: ankle_joint_1
  pid: {p: 7.0, i: 0.0, d: 0.2}

ankle_joint_3_position_controller:
  type: effort_controllers/JointPositionController
  joint: ankle_joint_3
  pid: {p: 7.0, i: 0.0, d: 0.2}

ankle_joint_5_position_controller:
  type: effort_controllers/JointPositionController
  joint: ankle_joint_5
  pid: {p: 7.0, i: 0.0, d: 0.2}

ankle_joint_2_position_controller:
  type: effort_controllers/JointPositionController
  joint: ankle_joint_2
  pid: {p: 7.0, i: 0.0, d: 0.2}

ankle_joint_4_position_controller:
  type: effort_controllers/JointPositionController
  joint: ankle_joint_4
  pid: {p: 7.0, i: 0.0, d: 0.2}

ankle_joint_6_position_controller:
  type: effort_controllers/JointPositionController
  joint: ankle_joint_6
  pid: {p: 7.0, i: 0.0, d: 0.2}
